<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/logo.png</normaloff>:/icon/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_task">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_requirement">
      <property name="text">
       <string>需要：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_layer">
      <property name="text">
       <string>层数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_time">
      <property name="text">
       <string>用时：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_hotend">
      <property name="text">
       <string>喷头：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_hotbed">
      <property name="text">
       <string>热床：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_box">
      <property name="text">
       <string>腔体：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
